release-1.8/release_notes_draft.md

```diff
@@ -102,6 +105,11 @@ kind.
  Users that have any code that depends on mutable selectors may continue to use 
  the apps/v1beta1 API for this release, but they should begin migration to code 
  that does depend on mutable selectors.
+ - Extended resources outside the `*kubernetes.io/` namespace are treated as
+ integers. Users can use any resource name of the form `[aaa.]my-domain.bbb/ccc`
+ in place of [Opaque Integer Resources](https://v1-6.docs.kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#opaque-integer-resources-alpha-feature).
+ These resources may not be overcommitted, so request and limit must be equal
```

Comment:
These resources cannot be overcommitted, so request and limit must be equal if both are present in a container spec.